src/test/java/org/jenkinsci/plugins/pipeline/milestone/MilestoneStepTest.java

```diff
@@ -177,6 +178,24 @@ public void milestoneNotAllowedInsideParallel() {
         });
     }
 
+    @Issue("JENKINS-38464")
+    @Test
```

Comment:
When reproducing a bug in a test without actually fixing it in the same commit, add

```java
@Ignore("TODO known to be broken")
```

Then the fix commit can remove the ignore.

May seem like a trivial point, but it is important when you start using `git bisect` etc.